frontends/common/options.cpp

```diff
@@ -154,6 +154,13 @@ CompilerOptions::CompilerOptions() : ParserOptions() {
             return true;
         },
         "Unrolling all parser's loops");
+    registerOption(
+        "--hsIndexSimplify", nullptr,
+        [this](const char*) {
+            hsIndexSimplify = true;
+            return true;
+        },
+        "Replace all non-cocrete indexes for header stacks in controls");
```

Comment:
This help message has a typo. But more importantly, I don't think it is clear.
Can you be more explicit?